internal/plugins/ansible/templates/roles/templates_dir.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package roles
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/constants"
+)
+
+const templatesPath = "templates" + constants.FilePathSep + ".placeholder"
```

Comment:
@asmacdo @fabianvf why is this placeholder file needed out of curiosity?

Reply:
This may not be required any more actually, this used to be so that the directory could be added to git but we don't do version control stuff anymore, right?

Reply:
or maybe it was just so that we could scaffold the empty dir, can't remember if that was possible or not

Reply:
>we don't do version control stuff anymore

Yeah, not anymore. If some other scaffolded file is going to be written in this directory, then this scaffold isn't necessary. If this is something that a user will eventually create manually, maybe documenting `mkdir ./templates` is better?

Reply:
The goal of the role scaffold was to mimic the output of `ansible-galaxy init`, which creates empty `files` and `templates` dirs. I'd prefer not to deviate from the established standard Ansible role directory scaffold

Reply:
IIRC this allows the empty `templates/`dir to be tracked in git,